app/pb_kits/playbook/pb_highlight/_highlight.jsx

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+/* eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+import Highlighter from 'react-highlight-words'
+import React from 'react'
+
+type HighlightProps = {
+  className?: String,
+  data?: String,
+  id?: String,
+  children?: Array<React.ReactNode>,
```

Comment:
This should be `children?: React.Node` instead.

Reply:
[Here's the reference documentation](https://flow.org/en/docs/react/children) for more info.